app/src/main/java/com/amaze/filemanager/utils/ServiceWatcherUtil.java

```diff
@@ -120,26 +155,26 @@ public static synchronized void runService(final Context context, final Intent i
         /*if (handlerThread==null || !handlerThread.isAlive()) {
```

Comment:
Commented code, if not needed, delete.

Reply:
Will delete later when I'll be working on a fix to the broken ````ProcessViewerFragment```` when returning to Amaze from progress notification.

Reply:
OK, but next time add a comment explaining that.